environment.yml

```diff
@@ -5,3 +5,5 @@ channels:
 dependencies:
   - mkdocs=1.3.0
   - mkdocs-material=8.5.2
+  - matplotlib
+  - pandas
```

Comment:
nit: missing end of file new line

can you specify versions for  matplotlib and pandas as well to help future proof this?

Reply:
I added versions and a newline!